### PR TITLE
Wip Add api bundle

### PR DESCRIPTION
**Pull request by AndyMJones**
Currently this is a working package that you can include to run in Browser and OpenFin. It also doesn't break Electron however you still need to globally install the Electron CLI.

This has been tested with notifications and a few other methods which all seem to be working as intended however it probably needs extensive testing.

### CODE REVIEW

File: packages/api-openfin/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "containerjs-api-openfin",
-  "version": "0.0.1",
+  "version": "0.0.2",
```
Comment (AndyMJones) [inline packages/api-openfin/package.json:3]: Had to update the version for npm to push the changes to the build folder

File: packages/api-openfin/src/shims.ts [hunk 2]
```diff
@@ -1,16 +1,22 @@
-window.Notification = function(title, options) {
-  if (!options) {
-    options = {};
-  }
+interface Window {
```
Comment (AndyMJones) [inline packages/api-openfin/src/shims.ts]: I don't like having this one interface in this file when all the others are in their own. However this is only needed for OpenFin and without it it throws the error that window doesn't have a fin property and without window the application throws an error when running in browser and Electron.

Comment (ColinEberhardt) [reply packages/api-openfin/src/shims.ts]: It's probably worth moving this comment, or something similar, into the `shim.ts` file itself.

**Review by ColinEberhardt — APPROVED** [review]
This all looks good to me - thanks :-)

Can you please updated the `README` in the root of ContainerJS to reference this bundle?

**Review by BenLambertNcl — APPROVED** [review]
LGTM!

**Pull request merged**